src/main.rs

```diff
@@ -157,6 +157,7 @@ async fn do_main() -> Result<()> {
 		&cfg.libp2p_psk_path,
 		libp2p_metrics,
 		cfg.libp2p_tcp_port_reuse,
+		(&cfg).into(),
```

Comment:
Since we are passing struct to `network::init`, can we remove other parameters and use them from struct?

Reply:
Well not for now, but in general - absolutely. The Kad parameters are logically separate from libp2p parameter, such as psk path and port reuse. In some next refactor iteration we'll def create an overarching network config structure that'll contain all of those...